src/model/Gender.enum.ts

```diff
@@ -0,0 +1,4 @@
+export enum Gender {
```

Comment:
> (I'm a random lurker trying to refer some friends to Mild Blue, but since this is a public repo, I took a look :) )
> 
> In particular, how is this better than / different from https://github.com/kub1x/rodnecislo?

@misslecter do we know this repo?